### DISCUSSION TITLE
Connection Error

### DISCUSSION THREAD

**Discussion by igorkrupitsky** [DaxStudio/DaxStudio#721] [General]
What might be the cause of this Power BI dataset connection error?

Authentication failed. We cannot locate Azure Active Directory in your environment. This could be because your client application does not currently have the required components. Install the latest version of your client application or contact technical support for assistance.

**Comment by dgosbell** [comment]
It's hard to say. That is not an error message we generate, that is coming from the Microsoft drivers. 

It could be due to running an old version of DAX Studio or it could be a typo in the username. 

What version of DAX Studio are you using? Are you connecting to a dataset in a Premium or PPU workspace? Is the XMLA Endpoint feature enabled? Can you connect with other client tools like Excel or SSMS?
